libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2569,6 +2569,9 @@ uint8_t GCS::get_channel_from_port_number(uint8_t port_num)
 
 MAV_RESULT GCS_MAVLINK::handle_command_request_message(const mavlink_command_long_t &packet)
 {
+    if (isnan(packet.param1)) {
```

Comment:
this would deny on commands that don't use any of the parameters, that doesn't seem reasonable